torch/testing/_internal/common_methods_invocations.py

```diff
@@ -298,6 +331,13 @@ def sample_inputs(self, device, dtype, requires_grad=False, **kwargs):
             samples = self.sample_inputs_func(self, device, dtype, requires_grad, **kwargs)
         except TypeError:
             samples = self.sample_inputs_func(self, device, dtype, requires_grad)
+
+        if 'include_conjugated_inputs' in kwargs and kwargs.get('include_conjugated_inputs'):
```

Comment:
Style nit: I think `include_conjugate_inputs` can be modeled as an explicit kwarg